src/main/java/com/khutircraftubackend/MarketPlaceBackendApplication.java

```diff
@@ -1,9 +1,11 @@
-package com.gmail.ypon2003.marketplacebackend;
+package com.khutircraftubackend;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableAsync;
 
 @SpringBootApplication
+@EnableAsync
 public class MarketPlaceBackendApplication {
```

Comment:
we don't want to rename it to Application, do we?

Reply:
This is name created automatically intellij idea. But we can change if needed.